crates/lib/test/fixtures/rules/std_rule_cases/LT06_tsql_declare.yml

```diff
@@ -0,0 +1,24 @@
+rule: LT06
```

Comment:
Can we drop this file? I think it's slightly redundant. 

We don't really want to add tests for every time the parser fails, that's really what the dialect tests are for.